kratos/processes/apply_multi_point_constraints_process.h

```diff
@@ -62,20 +63,25 @@ class ApplyMultipointConstraintsProcess : public Process
                 "constraint_set_name":"default",
                 "master_sub_model_part_name":"default_master",
                 "slave_sub_model_part_name":"default_slave",                
-                "variable_names":[""],
+                "variable_names":[],
                 "reform_every_step":false   
             }  )");
+        m_parameters.RecursivelyValidateAndAssignDefaults(default_parameters);
+
+        if(m_parameters["variable_names"].size() == 0)
```

Comment:
`KRATOS_ERROR_IF(m_parameters["variable_names"].size() == 0) << "In ApplyMultipointConstraintsProcess class constructor :: No DOFs specified for applying Multipoint constraints." << std::endl `